website/forms/features/cloudflare.md

```diff
@@ -3,6 +3,26 @@ id: cloudflare
 title: Cloudflare
 ---
 
-*Coming soon*
+## What is Cloudflare?
+
+Cloudflare is a large network of servers that can improve the security, performance, and reliability of anything connected to the Internet. Cloudflare does this by serving as a reverse proxy Open external link for your web traffic.
 
 ![Cloudflare screen](/img/forms/cloudflare.webp)
+
+## How do we use Cloudflare?
+
+Eightshift Forms doesn't configure the Cloudflare app or any other third-party tools. However, enabling this feature adds necessary configurations in the backend for everything to function correctly.
```

Comment:
```suggestion
After setting up Cloudflare, head to the Global settings and enable the Cloudflare integration. This will set up all the necessary options in the backend for everything to function properly.
```

Reply:
no an integration

Reply:
updated!